docs/index.html

```diff
@@ -196,7 +196,7 @@ <h3 class="section__subtitle">What's in UI Kit bundle?</h3>
                 alt="Sketch Logo"
                 class="section__logo"/>
 
-            <h3 class="section__item-title">Sketch UI Kit v0.9.0 </h3>
+            <h3 class="section__item-title">Sketch UI Kit v0.10.0 </h3>
             <p class="section__text">Our Sketch file has over 30 unique components refined and streamlined to allow any user the ability quickly and effeciently build Fiori 3 interfaces using the Fundamental Library.</p>
```

Comment:
**Quick edit**
Our Sketch file has over 30 unique components that are refined and streamlined to empower anyone using Fundamental Library with the ability to quickly and efficiently build Fiori 3 interfaces.